cl/users/templates/includes/webhooks_htmx/webhooks-test-webhook.html

```diff
@@ -0,0 +1,56 @@
+ <div class="row form-group">
+    <label class="col-md-3">Endpoint URL: </label><p class="col-md-9">{{ webhook.url }}</p>
+</div>
+
+<div class="row form-group">
+  <div class="col-xs-12">
+    <label for="dummy_data">Dummy Content:</label>
+    <div class="dummy-area">
+      <div class="copy-text" onclick="copy_text('dummy_data')"><i class="fa fa-copy"></i></div>
+      <textarea readonly class="form-control" id="dummy_data" rows="4">
+        {{ dummy_content }}
+      </textarea>
+    </div>
+  </div>
+</div>
+
+{% if da_type == webhook.event_type %}
+  <div class="row form-group">
+    <div class="col-xs-12" hx-ext="loading-states" data-loading-states>
+      <button id="webhook-send-test" class="btn btn-primary"
+              hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'
+              hx-post="{% url "webhooks-test-webhook" pk=webhook.id format='json' %}"
+              hx-target="#webhook-test-sent"
+              data-loading-target="#webhook-send-test"
+              data-loading-disable
+              >
+            Send Webhook Event Test
+            <i data-loading class="fa fa-spinner fa-spin"></i>
+      </button>
+      <span id="webhook-test-sent"></span>
+      <button id="view-in-logs-button" type="button" class="hidden btn btn-success">Sent it, view in Test Logs</button>
+    </div>
+  </div>
+{% endif %}
+
+<div class="row form-group">
+  <div class="col-xs-12">
+    <label for="dummy_curl">Test this Webhook event locally with the following <a data-toggle="collapse" href="#curl-command">CURL command:</a></label>
+    <div class="collapse" id="curl-command">
+      <div class="dummy-area">
+        <div class="copy-text" onclick="copy_text('dummy_curl')"><i class="fa fa-copy"></i></div>
+        <textarea readonly class="form-control" id="dummy_curl" rows="4">
+          {{ dummy_curl }}
+        </textarea>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
This needs to be a bit nicer. Maybe just use a second button next to the Test one that says, "Show cURL Command". When the button is clicked it shows the command as it does now, and below the input box it shows a little <p> tag that says:

> Use this command locally to replicate this event from our servers.

Reply:
Done, now it looks like this:

![Screen Shot 2022-11-14 at 13 46 55](https://user-images.githubusercontent.com/486004/201755350-c21bc484-7ebd-4fad-9dcc-d0570982e3a8.png)